README.md

```diff
@@ -1,3 +1,101 @@
+# Harbor with Rackspace Managed Kubernetes Auth
+
+This section explains how to create the development environment for developing the integration code for Harbor with Rackspace Managed Kubernetes Auth on Mac OSX. This uses your Docker/Docker Compose only because this is a local dev env. Integration with Kubernetes happens in [kubernetes-installer](https://github.com/rcbops/kubernetes-installer).
+
+Read the [Harbor](#harbor) section. Because master is unstable, all development happens in the [rackspace-mk8s-auth branch](https://github.com/rcbops/kubernetes-harbor/tree/rackspace-mk8s-auth) which is always based on a stable release branch. The rackspace-mk8s-auth branch will need to be rebased onto newer Harbor stable branches as we upgrade to newer releases.
+
+The only thing in the master branch is this section of the README just so it's easy to find.
+
+## Dev Env
+
+There are a few things you need to do to get a local dev env on Mac OSX going.
+
+1. Fork the repo.
+
+1. Even though the repo is git@github.com:rcbops/kubernetes-harbor.git it's easier to clone it to a directory structure where Go lang expects everything to be.
+
+    ```bash
+    mkdir $GOPATH/src/github.com/vmware
+    git clone git@github.com:my-github-username/kubernetes-harbor.git $GOPATH/src/github.com/vmware/harbor
```

Comment:
You have to specify the clone dir because if you left it as the default it would be named `kubernetes-harbor` which would mess with Go and its tooling.

Reply:
Ah okay 👍  didn't see that